README.md

```diff
@@ -26,9 +26,9 @@ See the [Raspberry Pi](README_RASPBERRY_PI.md) and [BeagleBone Black](README_BEA
 ### Platform support
 
 SuperCollider is tested with:
-- Windows 10 (32- and 64-bit) and MSVC 2017
-- macOS 10.13 with Xcode 9.4 and Xcode 10.1
-- Ubuntu 14.04 and gcc 4.9
+- Windows 10 (32- and 64-bit) and MSVC 2019
+- macOS 10.15 and Xcode 11.7
+- Ubuntu 18.04 and gcc 10
```

Comment:
hmm, we run a lot more than this in the build matrix -- are you only listing the jobs that actually run tests in CI?

Reply:
yeah... I wasn't sure what does "is tested with" mean in here, so I just updated information that was clearly outdated. And yes, for Linux I used the configuration that actually runs the test. 
Should I change this to something like
> Ubuntu 16.04-20.04, clang 6-10 and gcc 5-10

?

Reply:
yeah the meaning is not super clear. i think this works for now, maybe we could revisit it again during the release process and/or when moving to C++17